docs/source/tutorials/instrument-modeling/option-extension.rst

```diff
@@ -0,0 +1,77 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+How To Use the Option Extension Package
+#######################################
+
+To follow the script used in this tutorial, you can
+`clone the Daml Finance repository <https://github.com/digital-asset/daml-finance>`_. In particular,
+the Option test folder ``src/test/daml/Daml/Finance/Instrument/Option/Test/`` is the starting point
+of this tutorial.
+
+How To Create an Option Instrument
+**********************************
+
+In order to create an option instrument, you first have to decide what type of option you need. The
+:doc:`option extension package <../../packages/implementations/daml-finance-instrument-option>`
+currently supports the following types of options:
+
+European
+========
+
+The :ref:`European option <module-daml-finance-instrument-option-european-instrument-57671>`
+instrument models cash-settled, auto exercising call or put options. For reference, European options
+give the holder the right, but not the obligation, to buy (in case of a *call*) or to sell (in case
+of a *put*) the underlying asset at predetermined *strike* price on a specific *expiry* date in the
+future.
+
+As an example, we will create an option instrument that gives the holder the right to buy AAPL stock
+at a given price. This example is taken from
+``src/test/daml/Daml/Finance/Instrument/Option/Test/European.daml``, where all the details are
+available.
+
+We start by defining the terms:
+
+.. literalinclude:: ../../../../src/test/daml/Daml/Finance/Instrument/Option/Test/European.daml
+  :language: daml
+  :start-after: -- CREATE_EUROPEAN_OPTION_VARIABLES_BEGIN
+  :end-before: -- CREATE_EUROPEAN_OPTION_VARIABLES_END
+
+Now that we have defined the terms we can create the option instrument:
+
+.. literalinclude:: ../../../../src/test/daml/Daml/Finance/Instrument/Option/Test/Util.daml
+  :language: daml
+  :start-after: -- CREATE_EUROPEAN_OPTION_INSTRUMENT_BEGIN
+  :end-before: -- CREATE_EUROPEAN_OPTION_INSTRUMENT_END
+
+Once the instrument is created, you can book a holding on it using
```

Comment:
```suggestion
Once the instrument is created, you can create a holding on it using
```

Reply:
Also, you use the we form in the sentence above and then switch to the you form. We should probably keep it consistent